atomicapp/providers/kubernetes.py

```diff
@@ -38,227 +44,243 @@ class KubernetesProvider(Provider):
     This class implements deploy, stop and undeploy of an atomicapp on
     Kubernetes provider.
     """
+
+    # Class variables
     key = "kubernetes"
+    namespace = DEFAULT_NAMESPACE
+    k8s_artifacts = {}
+
+    # From the provider configuration
     config_file = None
-    kubectl = None
 
-    def init(self):
-        self.namespace = "default"
+    # Essential provider parameters
+    provider_api = "http://localhost:8080"
```

Comment:
the code below that uses this "default" value would actually make more sense if we set `localhost:8080` there instead of here. I would just set this to `None` unless there is a technical reason why we shouldn't. 

also, one could make an argument that this should be a constant??